test/Sema/exhaustive_switch_invocation_threshold.swift

```diff
@@ -0,0 +1,44 @@
+// Ensure that -switch-checking-invocation-threshold= frontend option works.
+//
+// RUN: not %target-swift-frontend -typecheck %s -switch-checking-invocation-threshold=1 2>&1 | %FileCheck %s --check-prefix LIMIT
+// RUN: not %target-swift-frontend -typecheck %s  2>&1 | %FileCheck %s --check-prefix NO_LIMIT
+// LIMIT: error: the compiler is unable to check that this switch is exhaustive in reasonable time
+// LIMIT-NOT: error: switch must be exhaustive
+// NO_LIMIT: error: switch must be exhaustive
+// NO_LIMIT-NOT: error: the compiler is unable to check that this switch is exhaustive in reasonable time
```

Comment:
This looks fine, but it's worth pointing out that `-NOT` lines only check that certain text does not appear after the previous positive check and before the next positive check. You could either bracket your `LIMIT`/`NO_LIMIT` lines with the `-NOT` checks, or save the error output to a file and check it twice (a common enough idiom in our test suite, look for `-check-prefix NEGATIVE`), or just not worry too much about it.